larch.js

```diff
@@ -90,6 +92,21 @@ function logMultiBackend(level, msg, meta, cb) {
     }
 };
 
+Larch.prototype.willSampleSingleBackend =
+function willSampleSingleBackend(level) {
+    return self.backends[0].willSample(level);
+};
+
+Larch.prototype.willSampleMultiBackend =
+function willSampleMultiBackend(level) {
```

Comment:
No, we want to take the log record if *any* backend wants it. The other ones will ignore it (as they told us they would) but the one who wanted it will get it. The optimization is to avoid allocating the metadata object when it isn't needed. If a single backend wants the record, we need to allocate that object.